### DISCUSSION TITLE
Weird PAUSE in the middle of the TTS audio

### DISCUSSION THREAD

**Discussion by abidhasanpintu** [deepgram/community#909] [General help]
I am using deepgram tts api to generate audio from text. But we are getting some weird PAUSE in the middle of the audio. We don't find any solution for that issue by looking at api docs. Can anyone check the attached audio and suggest what's the issue?

In the following audio after "on" a pause/break happens. I have used aura-stella-en voice

https://reisa.vsitesting3.com/storage/voice/5779e4a63788e9d113688c14ccc38e6c.mp3

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by abidhasanpintu** [reply]
Request Url: https://api.deepgram.com/v1/speak
Following curl request we are doing to get result:
```
curl_setopt_array($curl, array(
            CURLOPT_URL => 'https://api.deepgram.com/v1/speak?model=aura-stella-en&encoding=mp3&bitrate=32000',
            CURLOPT_RETURNTRANSFER => true,
            CURLOPT_ENCODING => '',
            CURLOPT_MAXREDIRS => 10,
            CURLOPT_TIMEOUT => 0,
            CURLOPT_FOLLOWLOCATION => true,
            CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
            CURLOPT_CUSTOMREQUEST => 'POST',
            CURLOPT_POSTFIELDS => $sayText,
            CURLOPT_HTTPHEADER => array(
                'Content-Type: text/plain',
                'Authorization: Token '.$deepgramApiKey
            ),
        ));
```

**Reply by jpvajda** [reply]
@abidhasanpintu  have you tried setting the container param yet? https://developers.deepgram.com/docs/tts-container. It can sometimes help with playback issues.

**Reply by abidhasanpintu** [reply]
@jpvajda Thanks for your response. We have tried container. But problem is, when we use container, we get bip/click sound at the beginning of the audio

**Comment by jpvajda** [comment]
Ok @abidhasanpintu , let me add our PM of TTS to see if she has some ideas for you @michellelychan .

**Comment by n-rane** [comment]
its due to article 'a' after 'on', not because of 'on'.
@deepgram any solution for this ??
